CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,132 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our project and community 
+a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, caste, color, religion, geographic location, or sexual
+identity and orientation.
```

Comment:
Should "political orientation" be mentioned in the list as well ?

Reply:
@NSeydoux usually not, as it's used as an escape hatch by the far-right when they harass people

Reply:
Oh I see. I figured it could be an issue, but my impression was that let's say banning a nazi for being a nazi is not harassement, but I see how that could be instrumentalized.